src/figuresClasses.ts

```diff
@@ -1,11 +1,97 @@
-export interface Figure {}
+enum Color {
+  red = 'red',
+  green = 'green',
+  blue = 'blue',
+}
+
+enum Shape {
+  triangle = 'triangle',
+  circle = 'circle',
+  rectangle = 'rectangle',
+}
+
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
+}
 
-export class Triangle implements Figure {}
+function getRoundedNumber(number: number): number {
+  return Math.floor(number * 100) / 100;
+}
+
+export class Triangle implements Figure {
+  public shape = Shape.triangle;
+
+  constructor(
+    public color: Color,
+    public a: number,
+    public b: number,
+    public c: number,
```

Comment:
In TypeScript, you don't need to manually assign constructor parameters to class properties if you use an access modifier. The properties 'color', 'a', 'b', and 'c' will be automatically created.